scripts/refresh-after-snapshot.py

```
@@ -0,0 +1,620 @@
+        raise RuntimeError(f"Could not get Keycloak admin token: {token_resp.stdout}")
+
+    realm = json.loads(Path(config.realm_json).read_text())
+    auth_header = f"Bearer {kc_token}"
+
+    for client in realm.get("clients", []):
+        if client.get("protocol") != "openid-connect":
+            continue
+        if client.get("publicClient") or client.get("bearerOnly"):
+            continue
+        uuid: str = client["id"]
+        resp = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)